referenceframe/frame.go

```diff
@@ -391,6 +420,19 @@ func (rf *rotationalFrame) Transform(input []Input) (spatial.Pose, error) {
 	return spatial.NewPoseFromOrientation(&spatial.R4AA{input[0].Value, rf.rotAxis.X, rf.rotAxis.Y, rf.rotAxis.Z}), err
 }
 
+// Interpolate interpolates the given amount between the two sets of inputs.
+func (rf *rotationalFrame) Interpolate(from, to []Input, by float64) ([]Input, error) {
```

Comment:
I think we should collapse the three functions in this file to a single one that operates on `baseFrame`

Reply:
I like that